Tests/swift-sdk-swift-tests/ClassExtensionsTests.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  ClassExtensionsTests.swift
+//  swift-sdk-swift-tests
+//
+//  Created by Jay Kim on 6/6/19.
+//  Copyright © 2019 Iterable. All rights reserved.
+//
+
+import XCTest
+
+@testable import IterableSDK
+
+class ClassExtensionsTests: XCTestCase {
+    func testUIColorInit() {
+        let blackColor = UIColor(red: 0, green: 0, blue: 0, alpha: 1)
+        
+        guard let hexColor = UIColor(hex: "000000") else {
```

Comment:
For clarity this should either be of the format "#000000" or 0x000000.

Reply:
I like that idea but the current initializer doesn't support it, but it could be modified in the future to accommodate those common forms of representation